decidim-admin/app/assets/javascripts/decidim/admin/sort_steps.js.es6

```diff
@@ -6,6 +6,13 @@
 // should have a `data-sort-url` attribute with the URL where the data should
 // be posted to.
 $(() => {
+ sortSteps();
+});
+$( document ).on('turbolinks:load' , function() {
+  sortSteps();
+});
+
+function sortSteps(){
```

Comment:
Consider defining this function as an expression like this: `const sortSteps = () => { ... }`. Then, since we are not using ES6 modules yet, you can "export" the method like this `window.sortSteps = sortSteps;`.